backend/resources/auth.py

```diff
@@ -32,38 +38,35 @@ def post(self):
         try:
             result = schema.load(request.json)
         except ValidationError as error:
-            return {"msg": "Wrong input data",
-                    "errors": error.messages}, HTTPStatus.BAD_REQUEST
+            return (
+                {"msg": "Wrong input data", "errors": error.messages},
+                HTTPStatus.BAD_REQUEST,
+            )
 
-        username = result['username']
-        password = result['password']
+        username = result["username"]
+        password = result["password"]
 
         if not (username and password):
-            return {"msg": "Username and password required"},
+            return ({"msg": "Username and password required"},)
```

Comment:
That's not a part of black introduction, but here
```
    return ({"msg": "Username and password required"},)
    HTTPStatus.BAD_REQUEST
```
the last line is never reachable and response sent to browser will have status set to `200`

Reply:
Fixed + removed unused import.